tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -3145,8 +3166,10 @@ def graAddresses(self, kernel, tP):
     tc = tP["tensorChar"]
     graIdx = 0
 
-    if tP["isA"] and tP["isSwizzled"]:
-      module.addModuleAsFlatItems(self.alignTo("StrideA0I", "StrideA0I", tP["swizzleK"]))
+    if tP["isSwizzled"]:
+      # "StrideA0I" or "StrideB1J"
+      strideName = "Stride%s%s"%(tc,self.states.indexChars[tP["idx"]])
+      module.addModuleAsFlatItems(self.alignTo(strideName, strideName, tP["swizzleK"]))
 
```

Comment:
strideName = "Stride%s%s"%(tc,self.states.indexChars[tP["idx"]])
this would be "StrideA0I" or "StrideB1J", thank Jimmy.